### PR TITLE
Update documents

### PR DESCRIPTION
**Pull request by okue**
#201 

README updated:
- modify URL links
- add description about `FlexSendMessage`
- add comments about passing **dict** to `FlexSendMessage#contents`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ multicast(self, to, messages, timeout=None)
 
 Send messages to multiple users at any time.
 
-https://developers.line.me/en/docs/messaging-api/reference/#send-multicast-messages
+https://developers.line.biz/en/reference/messaging-api/#send-multicast-messages
```
Comment (tkgauri) [inline README.rst]: https://developers.line.biz/en/reference/messaging-api/#send-multicast-message

File: README.rst [hunk 2]
```diff
@@ -748,6 +748,47 @@ TemplateSendMessage - ImageCarouselTemplate
...
+                url='https://example.com/cafe.jpg',
+                size='full',
+                aspect_ratio='20:13',
+                aspect_mode='cover',
+                action=URIAction(uri='http://example.com', label='label')
+            )
+        )
+    )
+
+※ You can pass a **dict** to FlexSendMessage#contents as follows:
```
Comment (cryeo) [inline README.rst]: [imo]
It seems that there are many properties which can accept dict value by using `Base#get_or_new_from_json_dict_with_types`.

How about informing it in more general place?

Comment (okue) [reply README.rst]: I recommend the use of **dict** only for `FlexSendMessage`.

Comment (cryeo) [reply README.rst]: OK. Then no problem.

**Pull request merged**